cmd/soci/commands/create.go

```diff
@@ -53,13 +55,26 @@ var CreateCommand = cli.Command{
 			Usage: "Minimum layer size to build zTOC for. Smaller layers won't have zTOC and not lazy pulled. Default is 10 MiB.",
 			Value: 10 << 20,
 		},
+		cli.StringSliceFlag{
+			Name:  optimizationFlag,
+			Usage: fmt.Sprintf("(Experimental) Enable optional optimizations. Valid values are %v", soci.Optimizations),
+		},
 	),
 	Action: func(cliContext *cli.Context) error {
 		srcRef := cliContext.Args().Get(0)
 		if srcRef == "" {
 			return errors.New("source image needs to be specified")
 		}
 
+		var optimizations []soci.Optimization
+		for _, o := range cliContext.StringSlice(optimizationFlag) {
```

Comment:
So the [v1](https://cli.urfave.org/v1/examples/flags/) documentation does not provide any guidance, but the [v2](https://cli.urfave.org/v2/examples/flags/#multiple-values-per-single-flag) documentation suggests it would be `soci --optimizations xattr --optimizations other`